Sources/PackageModel/ManifestSourceGeneration.swift

```diff
@@ -633,7 +633,7 @@ public struct SourceCodeFragment {
         self.subnodes = subnodes
     }
     
-    func generateSourceCode(indent: String = "") -> String {
+    public func generateSourceCode(indent: String = "") -> String {
```

Comment:
Would you mind changing all functions that were made `public` in this PR to `package` instead?

Reply:
Ah, sure! The only exception is likely to be the `AddPackageDependency` type itself, because I also want to use that from SourceKit-LSP.